src/gui/redux/model/model.js

```diff
@@ -1,10 +1,9 @@
-import { RECEIVED_DOCGEN } from '../docgen'
 import { UPDATE_PROP } from './model.actions'
 import { docgenToModel, convertFromGuiValue } from '../utils/docgen.js'
 
 export default (state = {}, { type, payload }) => {
   switch (type) {
-    case RECEIVED_DOCGEN:
+    case '@trampss/SET_DOCGEN':
```

Comment:
It would be great to have methods to get action names.
It will avoid that reducers will depends on the lib actions names (which can change in the future...à